driver-core/src/main/java/com/datastax/driver/core/PoolingOptions.java

```diff
@@ -400,30 +406,48 @@ public PoolingOptions setIdleTimeoutSeconds(int idleTimeoutSeconds) {
     }
 
     /**
-     * Returns the timeout when trying to acquire a connection from a host's pool.
-     *
-     * @return the timeout.
+     * @deprecated see {@link #setPoolTimeoutMillis(int)}. This method always returns 0.
      */
+    @Deprecated
     public int getPoolTimeoutMillis() {
-        return poolTimeoutMillis;
+        return 0;
+    }
+
+    /**
+     * @deprecated the connection pool does not use a timeout anymore, incoming requests are now throttled with a
+     * threshold on the {@link #setMaxQueueSize(int)}  queue size}. This method has no effect.
+     */
+    @Deprecated
+    public PoolingOptions setPoolTimeoutMillis(int poolTimeoutMillis) {
+        return this;
     }
 
     /**
-     * Sets the timeout when trying to acquire a connection from a host's pool.
+     * Returns the maximum number of requests that get enqueued if no connection is available.
+     *
+     * @return the maximum queue size.
+     */
+    public int getMaxQueueSize() {
```

Comment:
I wouldn't use "request" because it's too similar to "network request", but if you can think of something else I'm open to suggestions, it's true that maxQueueSize is not great.

I added a sentence to the javadoc of the setter as you suggested.